cmd/tracee-ebpf/main.go

```diff
@@ -158,6 +163,26 @@ func main() {
 				return fmt.Errorf("error creating Tracee: %v", err)
 			}
 
+			if listenMetrics {
+				err := metrics.RegisterEbpfEngineMetrics(t)
+				if err != nil {
+					if debug {
+						fmt.Fprintf(os.Stderr, "Error registering prometheus metrics: %v\n", err)
+					}
+				} else {
+					mux := http.NewServeMux()
+					mux.Handle("/metrics", promhttp.Handler())
+
+					go func() {
+						fmt.Fprintf(os.Stdout, "Serving metrics endpoint at %s\n", metricsAddr)
```

Comment:
This can be under debug condition (same in tracee-rules)

Reply:
There's no debug flag in tracee-rules, so this is based on the pprof endpoints there where this line is always logged.